envs/apache-beam-env.yaml

```diff
@@ -48,4 +48,4 @@ packages:
 {% endif %}
 {% endif %}
 
-git_tag_for_env: open-ce-v1.9.0
+git_tag_for_env: open-ce-v1.9.1
```

Comment:
These lines should be removed from all envs.

Reply:
yes, will add that change.

Reply:
Actually this is a direct porting PR. I will raise a follow up PR to remove the env strings.